rpc-client-api/src/response.rs

```diff
@@ -202,6 +202,7 @@ pub enum SlotUpdate {
         slot: Slot,
         timestamp: u64,
         stats: SlotTransactionStats,
+        hash: String,
```

Comment:
do we need to stringify this? why not just include as `Hash`?

Reply:
Yes, I agree - it would be more convenient for me as well. However, I looked at other subscriptions, RPC methods, and so on, and concluded that they use primitive types (for example, when retrieving a hash via an RPC method or when addresses in transactions are passed as strings rather than bytes). Apparently, this is a de facto standard in web programming. So, it will likely be more convenient for the vast majority.